src/bin/deku_node.ml

```diff
@@ -71,41 +85,62 @@ let handle_protocol_snapshot =
           last_block_signatures =
             Signatures.to_list snapshots.last_block_signatures;
         })
+
+(* POST /request-nonce *)
+(* Set a new nonce? *)
```

Comment:
Unused, but it set a new nonce for the key selected, so that the author can make a proof that he is actually the owner of the secret and so can set the URI.